tests/message_ut/message_ut.c

```diff
@@ -114,11 +123,8 @@ TEST_FUNCTION(message_create_succeeds)
 TEST_FUNCTION(message_create_2_times_yields_2_different_message_instances)
```

Comment:
There should be one test to check for general failures, using umock_c_negative_tests_*